bndtools.core/src/bndtools/wizards/bndfile/NewBndFileWizardPage.java

```diff
@@ -57,8 +57,14 @@ protected boolean validatePage() {
         IResource container = ResourcesPlugin.getWorkspace().getRoot().findMember(containerPath);
         IProject project = container.getProject();
 
-        if (Project.BNDFILE.equalsIgnoreCase(fileName)) {
-            error = Messages.NewBndFileWizardPage_errorReservedFilename;
+        try {
+            if (project.hasNature(BndtoolsConstants.NATURE_ID)) {
+                if (Project.BNDFILE.equalsIgnoreCase(fileName)) {
+                    error = Messages.NewBndFileWizardPage_errorReservedFilename;
```

Comment:
`bndtools.wizards.bndfile.NewBndFileWizardPage` - You really won't see it unless you are try to add  a `bnd.bnd` file to a non-Bndtools project. Easy enough to change to something more friendly if you have any ideas?